packages/blockchain-wallet-v4-frontend/src/data/components/swap/utils.ts

```diff
@@ -34,6 +34,29 @@ export const getPair = (BASE: SwapAccountType, COUNTER: SwapAccountType) => {
   return `${BASE.coin}-${COUNTER.coin}`
 }
 
+export const interpolatePrice = (
+  thisVol: BigNumber,
+  thisPrice: BigNumber,
+  nextVol: BigNumber,
+  nextPrice: BigNumber,
+  amount: BigNumber
+): number | string => {
+  try {
+    // if (thisPrice.isGreaterThan(nextPrice)) throw new Error('Should be sorted')
+    // if (thisVol.isGreaterThan(nextVol)) throw new Error('Should be sorted')
+    // if (amount.isLessThan(thisVol) && amount.isGreaterThan(nextVol))
+    //   throw new Error('Amount should be between x[0] and x[1]')
```

Comment:
what about removing this one?

Reply:
this popped up because of some linter changes that I did. I'll remove it though.